mackerel-plugin-sidekiq/lib/sidekiq.go

```diff
@@ -141,6 +144,42 @@ func (sp SidekiqPlugin) getEnqueued() uint64 {
 	return inject(queuesLlens, 0)
 }
 
+func (sp SidekiqPlugin) getLatency() uint64 {
+	key := addNamespace(sp.Namespace, "queues")
+	queues := sp.sMembers(key)
+	queuesLatencies := make([]uint64, 10)
+
+	prefix := addNamespace(sp.Namespace, "queue:")
+	for _, e := range queues {
+                queuesLRange, err := sp.Client.LRange(prefix+e, -1, -1).Result()
+                if err != nil {
+                        fmt.Fprintf(os.Stderr, "get last queue error")
+                }
+
+                if len(queuesLRange) == 0 {
+                        return 0
+                }
+                var job map[string]interface{}
+                var thence int64
+
+                err = json.Unmarshal([]byte(queuesLRange[0]), &job)
+                if err != nil {
+                        fmt.Fprintf(os.Stderr, "json parse error")
+                        return 0
+                }
+                now := time.Now().Unix()
+                if enqueuedAt, ok := job["enqueued_at"]; ok {
+                        enqueuedAt := enqueuedAt.(float64)
+                        thence = int64(enqueuedAt)
+                } else {
+                        thence = now
+                }
+		queuesLatencies = append(queuesLatencies, uint64(now - thence))
+	}
+
+	return inject(queuesLatencies, 0)
```

Comment:
I thought it would be more useful to be able to check the delay for each queue, so I changed the implementation.
Because how much delay is acceptable for each queue depends on the type of job.